modules/navier_stokes/test/tests/finite_volume/pwcns/channel-flow/gold/2d-transient-gas_out.csv

```diff
@@ -0,0 +1,5 @@
+time,inlet-p,outlet-temp,outlet-u,solid-temp
+0,0,0,0,0
+1,8.777543485164,110.76196226955,-2.7731418339086,107.52928262687
```

Comment:
side average value clearly not the way to get velocities :O

Reply:
wow. Are we going to keep this gold file?

Reply:
yeah actually if you look at the exodus the gas moves weird on the first step
changed to VolFlowRate

Reply:
interesting